doc/source/whatsnew/v0.19.0.txt

```diff
@@ -289,6 +289,10 @@ Other enhancements
 
     pd.Timestamp(year=2012, month=1, day=1, hour=8, minute=30)
 
+
+- ``pd.read_csv()`` with the ``chunksize=`` option and implicit index now returns an index progressively numbered, rather than in repeated chunks (:issue:`12185`)
```

Comment:
show an example of the before and the after, this is a major change. put it in a separate sub-section.